src/TypeSpec.Extension/Emitter.Csharp/test/Unit/model-type.test.ts

```diff
@@ -69,13 +69,12 @@ op test(@body input: Pet): Pet;
                 SerializedName: "kind",
                 Type: {
                     Kind: "string",
-                    IsNullable: false,
                     Encode: undefined
                 },
                 IsRequired: true,
                 IsReadOnly: false,
                 IsDiscriminator: true,
-                Description: "Discriminator",
```

Comment:
why this changes?

Reply:
TCGC update the description of `discriminator` in the latest version. So there are a lot of description changes in xml doc.